platform/darwin/src/MGLRasterSource.h

```diff
@@ -104,6 +104,109 @@ MGL_EXPORT
  */
 - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL tileSize:(CGFloat)tileSize NS_DESIGNATED_INITIALIZER;
 
+/**
+ Returns a raster source initialized an identifier, tile URL templates, and
+ options.
+
+ After initializing and configuring the source, add it to a map view’s style
+ using the `-[MGLStyle addSource:]` method.
+
+ #### Tile URL templates
```

Comment:
There's currently [no indication](https://www.mapbox.com/ios-sdk/api/3.5.0/Classes/MGLVectorSource.html) in the documentation of how to initialize `MGLVectorSource`. I think this is an improvement, despite the duplication.